server/scraper.py

```diff
@@ -0,0 +1,54 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+headers = {
+    'Accept':
+        'text/html,application/xhtml+xml,application/xml;q=0.9,image/'
+        'webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7',
+    'User-Agent':
+        'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 '
+        '(KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36 Edg/120.0.0.0'
+}
+
+
+def scrap_from_search(search: str) -> list[dict]:
+    link_src = f'https://www.laptopmag.com/search?searchTerm={search}&articleType=best-pick'
+    page_src = requests.get(link_src, headers)
+    soup_src = BeautifulSoup(page_src.content, 'html.parser')
+
+    laptops = soup_src.find_all('div', class_='listingResult')
+    laptops_data = []
+    for laptop in laptops:
+        dct = dict()
+        dct['link'] = laptop.find('a', class_='article-link').get('href')
+        dct['image'] = laptop.find('img').get('data-pin-media')
+        dct['title'] = laptop.find('h3', class_='article-name').text.strip()
+        dct['author'] = laptop.find('span', attrs={'style': 'white-space:nowrap'}).text.strip()
```

Comment:
The line of code must not be greater than 79 symbols.

Reply:
We will talk about it next on our next meeting.